<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QWidget" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExpertEstimationMethodLabs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,4">
     <item>
      <widget class="QSpinBox" name="expert_number">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="alternative_number">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,4">
     <item>
      <widget class="QTableWidget" name="experts">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>50</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>50</number>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="alternatives">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>50</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>50</number>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="estimations">
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>50</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>50</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="result_layout"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
